src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateLegalEntityCommandTests/WhenICallCreateLegalEntity.cs

```diff
@@ -228,7 +231,10 @@ public async Task ThenAddedLegalEntityEventIsPublished()
                 e.AccountLegalEntityPublicHashedId.Equals(ExpectedAccountLegalEntityPublicHashString) &&
                 e.OrganisationName.Equals(_command.Name) &&
                 e.UserName.Equals(_owner.FullName()) &&
-                e.UserRef.Equals(Guid.Parse(_owner.UserRef)))));
+                e.UserRef.Equals(Guid.Parse(_owner.UserRef)) &&
+                e.OrganisationReferenceNumber.Equals(_agreementView.LegalEntityCode) &&
+                e.OrganisationAddress.Equals(_agreementView.LegalEntityAddress) &&
+                e.OrganisationType.ToString().Equals(_agreementView.LegalEntitySource.ToString())))); 
             //e.Created.)));
```

Comment:
Can we remove this commented line?

Reply:
Addressed by https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/1556/commits/d4498ca99b190f639df1175d410c63e7aeb104ba